platform/api/oci/instance.go

```diff
@@ -35,10 +35,15 @@ func (a *API) CreateInstance(name, userdata string) (*Machine, error) {
 		return nil, err
 	}
 
+	metadata := map[string]string{
+		"user_data": base64.StdEncoding.EncodeToString([]byte(userdata)),
```

Comment:
Come to think of it, could you add another commit which sets this conditionally as well?  Users will probably not explicitly set a zero-length `user_data` field.

Reply:
New commit pushed addressing this.